wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSenderMessageDetailsCell.swift

```diff
@@ -168,88 +160,104 @@ class ConversationSenderMessageDetailsCell: UIView, ConversationMessageCell {
         ])
     }
 
-    private func configureAuthorLabel(object: Configuration, user: UserType, fullName: String) {
-        let textColor: UIColor = user.isServiceUser ? SemanticColors.Label.textDefault : user.accentColor
-        var attributedString = NSMutableAttributedString(
-            string: fullName,
+    private func configureAuthorLabel(object: Configuration) {
+        let textColor: UIColor = object.user.isServiceUser ? SemanticColors.Label.textDefault : object.user.accentColor
+        let attributedString = NSMutableAttributedString(
+            string: object.user.name ?? "",
```

Comment:
in which case do we get no name? is it in the case backend offline or something I remember @KaterinaWire Pr showing a default string

Reply:
I don't see a default value. I was just checking the codebase and I see the same pattern as the one I wrote:

`viewModel.user.name ?? ""`